app/components/meeting/edit-meeting-modal.js

```diff
@@ -140,11 +147,12 @@ export default class MeetingEditMeetingComponent extends Component {
   }
 
   initializeSecretary = task(async () => {
-    if (this.enableDigitalAgenda) {
+    if (this.enableDigitalAgenda && !this.isPreKaleidos) {
       const secretary = await this.args.meeting.secretary;
       if (isPresent(secretary)) {
         this.secretary = secretary;
-      } else {
+      } else if (this.isNew) {
+        // if a meeting had no secretary yet we don't set a default one automatically
```

Comment:
This needs feedback from the users.
I think we should not set a default on editing meeting since that meeting.
When you edit older meetings (before the release of the secretary feature than that meeting + all agendaitems will suddenly get a secretary

Reply:
That's perfect